srcs/Config/LocationConfig/LocationConfig.cpp

```diff
@@ -0,0 +1,280 @@
+#include "LocationConfig.hpp"
+
+LocationConfig::LocationConfig() {
+	init_location_keyword();
+}
+
+LocationConfig::LocationConfig(const LocationConfig &other)
+{
+	this->_autoindex = other.get_autoindex();
+	this->_chunked_transferencoding_allow = other.get_chunked_transferencoding_allow();
+	this->_server_tokens = other.get_server_tokens();
+	this->_client_body_buffer_size = other.get_client_body_buffer_size();
+	this->_client_body_timeout = other.get_client_body_timeout();
+	this->_client_header_buffer_size = other.get_client_header_buffer_size();
+	this->_client_header_timeout = other.get_client_header_timeout();
+	this->_client_max_body_size = other.get_client_max_body_size();
+	this->_keepalive_requests = other.get_keepalive_requests();
+	this->_keepalive_timeout = other.get_keepalive_timeout();
+	this->_alias = other.get_alias();
+	this->_accesslog = other.get_accesslog();
+	this->_cgi_path = other.get_cgi_path();
+	this->_default_type = other.get_default_type();
+	this->_errorlog = other.get_errorlog();
+	this->_upload_path = other.get_upload_path();
+	this->_root = other.get_root();
+	this->_allow_methods = other.get_allow_methods();
+	this->_index = other.get_index();
+	this->_server_name = other.get_server_name();
+	this->_errorpages = other.get_errorpages();
+}
```

Comment:
```suggestion
LocationConfig::LocationConfig(const LocationConfig &other)
{
	*this = other;
}
```

`operator=`活用して、これで十分かも

Reply:
確かに、修正済み